hawkular-alerts-engine/src/test/java/org/hawkular/alerts/engine/DefinitionsTest.java

```diff
@@ -72,51 +78,538 @@ public void test000InitScheme() throws Exception {
     }
 
     @Test
-    public void test001CopyTrigger() throws Exception {
-        Trigger t = definitionsService.getTrigger(TEST_TENANT, "trigger-1");
-        assertTrue(t != null);
+    public void test0010GroupTrigger() throws Exception {
```

Comment:
To be honest, I'm not even sure these tests are performed in order. I'm not sure the superclass' annotation is getting picked up. But +1 either way.